src/neuroconv/nwbconverter.py

```diff
@@ -257,28 +258,15 @@ def get_default_backend_configuration(
 
 
 class ConverterPipe(NWBConverter):
-    """Takes a list or dict of pre-initialized interfaces as arguments to build an NWBConverter class"""
-
-    def get_conversion_options_schema(self) -> dict:
```

Comment:
Same issue as above; odd that this is the first method defined even before any class methods or `__init__`